python/ray/_private/runtime_env/context.py

```diff
@@ -81,6 +81,13 @@ def exec_worker(self, passthrough_args: List[str], language: Language):
             passthrough_args[0] = default_worker_path
 
         if sys.platform == "win32":
+
+            def quote(s):
+                s = s.replace("&", "%26")
```

Comment:
`&` definitely needs escaping because it breaks the argument processing.

`?` does not need escaping (it appears in the test which is passing). I don't know what other non-ascii characters are valid?

Reply:
I opened #44180 to track other characters. It would be nice to get this in since it appears in tests.